app/src/main/java/org/mozilla/fenix/historymetadata/HistoryMetadataFeature.kt

```diff
@@ -72,3 +79,6 @@ class HistoryMetadataFeature(
         job?.cancel()
     }
 }
+
+// The last updated time of the group is based on the most recently updated item in the group
+fun HistoryMetadataGroup.lastUpdated(): Long = historyMetadata.maxOf { it.updatedAt }
```

Comment:
Can we move this into HistoryMetadataGroup.kt instead?

Reply:
👍 yes, moved it over.